src/applications/coronavirus-vaccination-expansion/config/form.js

```diff
@@ -20,38 +21,33 @@ import manifest from '../manifest.json';
 const formConfig = {
   rootUrl: manifest.rootUrl,
   urlPrefix: '/',
-  // submitUrl: '/v0/api',
+  submitUrl: `${environment.API_URL}/covid_vaccine/v0/expanded_registration`,
   submit: () =>
```

Comment:
I believe you need to remove this `submit` function on line 25 as it is overriding the default submit behavior and just returning a mocked response.